test/application/use.js

```diff
@@ -70,60 +65,50 @@ describe('app.use(fn)', () => {
 
     const server = app.listen();
 
-    request(server)
-      .get('/')
-      .expect(404)
-      .end(err => {
-        if (err) return done(err);
-        calls.should.eql([1, 2, 3, 4, 5, 6]);
-        done();
-      });
+    const res = await request(server, '/');
+    expect(res.status).toBe(404);
```

Comment:
Hmm since we've managed to avoid transformations thus far, I'm not sure we'd want to add one. However, I agree that it'd be very useful.